dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentOperationParamsMapperTest.java

```diff
@@ -186,7 +185,7 @@ void shouldMapOrgUnitsWhenOrgUnitUidsAreSpecified()
   }
 
   @Test
-  void shouldThrowExceptionWhenOrgUnitNotFound() {
+  void shouldFailWhenOrgUnitNotFound() {
     EnrollmentOperationParams operationParams =
         EnrollmentOperationParams.builder()
```

Comment:
I think ... shouldFail ... is a bit generic here, it could be more like .. forbiddenAccesWhen ... or ... shouldThrowForbiddenException ...

Reply:
I don't really have a strong opinion on this, I just did it because in other tests we already had "shouldFail", so I wanted to harmonize this. But I'm fine with having a more explicit test name. I'll change it back.